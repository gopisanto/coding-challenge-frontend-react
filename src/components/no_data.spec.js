import React from 'react';
import renderer from 'react-test-renderer';

import NoData from './no_data';

it('renders correctly', () => {
  const component = renderer.create(<NoData />).toJSON();

  expect(component).toMatchSnapshot();
});

it('renders correctly when custom message passed', () => {
  const component = renderer.create(<NoData message="No records found." />).toJSON();

  expect(component).toMatchSnapshot();
});
