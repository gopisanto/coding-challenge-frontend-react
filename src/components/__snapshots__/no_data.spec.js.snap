// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  No data found
</div>
`;

exports[`renders correctly when custom message passed 1`] = `
<div>
  No records found.
</div>
`;
